cant run example because of error

Hi,
I just cloning this example and run `flutter run` but found below error :

```
zukinarus-Air:example zukinaru$ flutter run
Launching lib/main.dart on SM N9208 in debug mode...
Initializing gradle...                                              6.3s
Resolving dependencies...                                          14.7s
                                                                        
Compiler message:                                                       
lib/ui/frienddetails/footer/skills_showcase.dart:27:23: Error: 'RangeSlider' is imported from both 'package:flutter/src/material/range_slider.dart' and 'package:flutter_range_slider/src/flutter_range_slider.dart'.
                      RangeSlider(                                      
                      ^^^^^^^^^^^                                       
lib/ui/frienddetails/footer/skills_showcase.dart:43:17: Error: 'RangeSlider' is imported from both 'package:flutter/src/material/range_slider.dart' and 'package:flutter_range_slider/src/flutter_range_slider.dart'.
                RangeSlider(                                            
                ^^^^^^^^^^^                                             
lib/ui/frienddetails/footer/skills_showcase.dart:58:17: Error: 'RangeSlider' is imported from both 'package:flutter/src/material/range_slider.dart' and 'package:flutter_range_slider/src/flutter_range_slider.dart'.
                RangeSlider(                                            
                ^^^^^^^^^^^                                             
Compiler failed on /Users/zukinaru/zipediaPrj/mobile/highlighter-coachmark/example/lib/main.dart
                                                                        
FAILURE: Build failed with an exception.                                
                                                                        
* Where:                                                                
Script '/Users/zukinaru/development/flutter/packages/flutter_tools/gradle/flutter.gradle' line: 665
                                                                        
* What went wrong:                                                      
Execution failed for task ':app:compileflutterBuildDebugArm64'.         
> Process 'command '/Users/zukinaru/development/flutter/bin/flutter'' finished with non-zero exit value 1
                                                                        
* Try:                                                                  
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
                                                                        
* Get more help at https://help.gradle.org                              
                                                                        
BUILD FAILED in 12s                                                     
Running Gradle task 'assembleDebug'...                                  
Running Gradle task 'assembleDebug'... Done                        13.8s
Gradle task assembleDebug failed with exit code 1
```

Any Idea to solve ?

Thanks In Advance...

--- a/example/lib/ui/frienddetails/footer/skills_showcase.dart
+++ b/example/lib/ui/frienddetails/footer/skills_showcase.dart
@@ -1,6 +1,6 @@
 import 'package:flutter/material.dart';
 
-import 'package:flutter_range_slider/flutter_range_slider.dart';
+import 'package:flutter_range_slider/flutter_range_slider.dart' as rngslider;
 
 class SkillsShowcase extends StatelessWidget {
   static final GlobalKey sliderKey = GlobalObjectKey("sliderKey");
@@ -24,7 +24,7 @@ class SkillsShowcase extends StatelessWidget {
                         style: textTheme.body1
                             .copyWith(color: Colors.white70, fontSize: 16.0)))
                     ..add(
-                      RangeSlider(
+                      rngslider.RangeSlider(
                         min: 0.0,
                         max: 100.0,
                         lowerValue: 0.0,
@@ -40,7 +40,7 @@ class SkillsShowcase extends StatelessWidget {
                   style: textTheme.body1
                       .copyWith(color: Colors.white70, fontSize: 16.0)))
               ..add(
-                RangeSlider(
+                rngslider.RangeSlider(
                   min: 0.0,
                   max: 100.0,
                   lowerValue: 0.0,
@@ -55,7 +55,7 @@ class SkillsShowcase extends StatelessWidget {
                   style: textTheme.body1
                       .copyWith(color: Colors.white70, fontSize: 16.0)))
               ..add(
-                RangeSlider(
+                rngslider.RangeSlider(
                   min: 0.0,
                   max: 100.0,
                   lowerValue: 0.0,

